jobs/golangapiserver/spec

```diff
@@ -156,6 +156,18 @@ properties:
   autoscaler.apiserver.scaling_rules.cpuutil.upper_threshold:
     description: "Allowable upper threshold of the cpuutil scaling range"
     default: 100
+  autoscaler.apiserver.scaling_rules.diskutil.lower_threshold:
+    description: "Allowable lower threshold of the diskutil scaling range"
+    default: 1
+  autoscaler.apiserver.scaling_rules.diskutil.upper_threshold:
+    description: "Allowable upper threshold of the diskutil scaling range"
+    default: 100
+  autoscaler.apiserver.scaling_rules.disk.lower_threshold:
+    description: "Allowable lower threshold of the disk scaling range"
+    default: 1
+  autoscaler.apiserver.scaling_rules.disk.upper_threshold:
+    description: "Allowable upper threshold of the disk scaling range"
+    default: 8000
```

Comment:
Just curios, where do we get this magic number ?

Reply:
Good catch! 

I changed this to be the maximum CF disk quota: [277bdd4](https://github.com/cloudfoundry/app-autoscaler-release/pull/2811/commits/277bdd4950b3b462c3fed9ef706cf9df0c243609)